app/processing/statement/get-statement.js

```diff
@@ -0,0 +1,26 @@
+const { getDetails, getAddress, getDetailedFunding, getScheme, getDetailedPayments } = require('./components')
+const getCalculation = require('../calculation')
+const getPaymentRequest = require('../payment-request')
+const getSettlement = require('../settlement')
+
+const getStatement = async (settlementId) => {
+  const settlement = await getSettlement(settlementId)

```

Comment:
This need a `try-catch` -- does this ticket cover the 'what to do once a statement schedule cannot progress'?